### DISCUSSION TITLE
I can't make "queue" work

### DISCUSSION THREAD

**Discussion by tbordinat** [owen-it/laravel-auditing#936] [Q&A] [answered]
Hi !

I'm using **v13.6.5** of this package and I'm having an issue with queues.

I configured my `audit.php` file like this : 

```php
    'queue' => [
        'enable'     => true,
        'connection' => 'database',
        'queue'      => 'default',
        'delay'      => 0,
    ],
```

With the following `queue.php`

```
php
    'connections' => [
        'database' => [
            'driver' => 'database',
            'table' => 'jobs',
            'queue' => 'default',
            'retry_after' => 1000, // more than the maximum timeout
            'after_commit' => false,
        ],
    ],
```

I can't make it work locally.
When I disable queue (`enable => false`), it works like a charm, model events are stores in my `audits` table.
My queues work, I've got my jobs table and events are stored / executed properly.
But since à enable queuing in audit.php, nothing, `jobs` stays empty and nothing happen (no error, no log, nothing). 

![image](https://github.com/owen-it/laravel-auditing/assets/4682556/390ddf2e-660a-4d58-8771-4af4b0216dbd)

Do I have to listen to the events manually?

There is nothing about the queues in the documentation https://laravel-auditing.com/guide

**Comment by erikn69** [comment, marked as answer]
- Hi try https://github.com/owen-it/laravel-auditing/pull/893 on dev branch
- Look at the `jobs`, `jobs_failed` tables
- Is the Queue Worker running? [laravel.com/queues#running-the-queue-worker](https://laravel.com/docs/11.x/queues#running-the-queue-worker)
- Also look your default queue https://github.com/owen-it/laravel-auditing/blob/0ec27626dc0464dbada9422bb89b1de06954126b/src/Auditor.php#L90-L92
